homework_unit_14/shallow_menu/main.js

```diff
@@ -0,0 +1,156 @@
+const shallowMenu = [
+  {
+    title: 'Dashboard',
+    link: '/dashboard',
+    children: [
+      {
+        title: 'Tool',
+        link: '/dashboard/tool'
+      },
+      {
+        title: 'Reports',
+        link: '/dashboard/reports'
+      },
+      {
+        title: 'Analytics',
+        link: '/dashboard/analytics'
+      },
+      {
+        title: 'Code Blocks',
+        link: '/dashboard/code-blocks'
+      },
+    ]
+  },
+  {
+    title: 'Sales',
+    link: '/sales',
+    children: [
+      {
+        title: 'New Sales',
+        link: '/sales/new-sales'
+      },
+      {
+        title: 'Expired Sales',
+        link: '/sales/expired-sales'
+      },
+      {
+        title: 'Sales Reports',
+        link: '/sales/sales-reports'
+      },
+      {
+        title: 'Deliveries',
+        link: '/sales/deliveries'
+      },
+    ]
+  },
+  {
+    title: 'Messages',
+    link: '/messages',
+    children: [
+      {
+        title: 'Inbox',
+        link: '/messages/inbox'
+      },
+      {
+        title: 'Outbox',
+        link: '/messages/outbox'
+      },
+      {
+        title: 'Sent',
+        link: '/messages/sent'
+      },
+      {
+        title: 'Archived',
+        link: '/messages/archived'
+      },
+    ]
+  },
+  {
+    title: 'Users',
+    link: '/users',
+    children: [
+      {
+        title: 'New User',
+        link: '/users/new-user'
+      },
+      {
+        title: 'User Groups',
+        link: '/users/user-groups'
+      },
+      {
+        title: 'Permissions',
+        link: '/users/permissions'
+      },
+      {
+        title: 'passwords',
+        link: '/users/passwords'
+      },
+    ]
+  },
+  {
+    title: 'Settings',
+    link: '/settings',
+    children: [
+      {
+        title: 'Databases',
+        link: '/settings/databases'
+      },
+      {
+        title: 'Design',
+        link: '/settings/design'
+      },
+      {
+        title: 'Change User',
+        link: '/settings/change-user'
+      },
+      {
+        title: 'Log Out',
+        link: '/settings/log-out'
+      },
+    ]
+  },
+]
+
+function renderCodeHtml () {
+  const menuList = document.querySelector('.menu__list')
+  menuList.innerHTML = shallowMenu.map(value => {
+  const [a,b,c,d] = value.children
```

Comment:
```suggestion
  const [a, b, c, d] = value.children
```
`a b c d` ????
Trong mỗi menu không phải sẽ luôn có 4 children, có thể không có cái nào, có thể sẽ có 10 cái, cũng có thể có 11 cái